deploy/docker/seqr/entrypoint.sh

```diff
@@ -35,6 +35,16 @@ if [ $SEQR_GIT_BRANCH ]; then
   git checkout $SEQR_GIT_BRANCH
 fi
 
+# init seqrdb unless it already exists
+if ! psql --host postgres -U postgres -l | grep seqrdb; then
+
+  psql --host postgres -U postgres -c 'CREATE DATABASE seqrdb';
+  psql --host postgres -U postgres -c 'CREATE DATABASE reference_data_db';
+  psql --host postgres -U postgres reference_data_db <  <(curl -s https://storage.googleapis.com/seqr-reference-data/gene_reference_data_backup.gz | gunzip -c -);
```

Comment:
I would strongly prefer if we use a variable to reference the backup instead of a hardcoded string here

Reply:
This path will remain the same for all installs. Are you suggesting a variable defined at the top of this script?
Also, at some point this file should have a version.

Reply:
Yeah I just meant a variable at the top of the script

Reply:
I mean ideally I'd rather it be a kubernetes variable and we could just set it in the shared YAML, but at a minimum a variable in the script would be nice